specification/servicefabric/resource-manager/readme.md

```diff
@@ -21,22 +21,53 @@ To see additional help and options, run:
+### Basic Information 
+These are the global settings for the ServiceFabricManagementClient API.
 
 ``` yaml
+title: ServiceFabricManagementClient
+description: Service Fabric Management Client
 openapi-type: arm
-tag: package-2017-07
-```
+tag: package-2018-02
+
+directive:
+  - suppress: ListInOperationName
+    reason: Modifying the operation names would break the backwards compatibility of the API.
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+  - suppress: SummaryAndDescriptionMustNotBeSame
+    reason: There are a lot of APIs with missing summary content. While it is being worked on disabling this to ensure that we catch and fix other violations.
+  - suppress: TrackedResourceListByImmediateParent
+    reason: Proxy resources are not properly evaluated by the validation toolset.
+  - suppress: DefinitionsPropertiesNamesCamelCase
+    reason: Modifying the operation names would break the backwards compatibility of the API.
+  - suppress: EnumInsteadOfBoolean
+    reason: The boolean properties are actually boolean value in the Service Fabric's application model.
+  - suppress: TrackedResourceGetOperation
+    reason: Proxy resources are not properly evaluated by the validation toolset.
+  - suppress: TrackedResourcePatchOperation
+    reason: Proxy resources are not properly evaluated by the validation toolset.
+  - suppress: TrackedResourceListByResourceGroup
+    reason: Proxy resources are not properly evaluated by the validation toolset.
+  - suppress: TrackedResourceListBySubscription
+    reason: Proxy resources are not properly evaluated by the validation toolset.
+  - suppress: DescriptionAndTitleMissing
+    reason: There are a lot of APIs with missing titles. While it is being worked on disabling this to ensure that we catch and fix other violations.
+  - suppress: Example Validations
+    reason: There are open issues (bugs) in the validator affecting some of the examples and since there is no way to selectively disable the validation for a particular example or paths, all of the example validation is being turned off.
+  - suppress: Example Validations
+    reason: There are open issues (bugs) in the validator affecting some of the examples and since there is no way to selectively disable the validation for a particular example or paths, all of the example validation is being turned off.
 
+```
 
-### Tag: package-2016-09
+### Tag: package-2018-02
 
-These settings apply only when `--tag=package-2016-09` is specified on the command line.
+These settings apply only when `--tag=package-2018-02` is specified on the command line.
 
-``` yaml $(tag) == 'package-2016-09'
+``` yaml $(tag) == 'package-2018-02'
 input-file:
-- Microsoft.ServiceFabric/stable/2016-09-01/servicefabric.json
+- Microsoft.ServiceFabric/stable/2018-02-01/cluster.json
+- Microsoft.ServiceFabric/preview/2017-07-01-preview/application.json

```

Comment:
In general this PR looks pretty much textbook, and that is much appreciated!

I did want to point out for the sake of awareness, that referencing these common definitions will result in the clients calling the service with both API Versions, as seen here in the Go SDK:
https://github.com/Azure/azure-sdk-for-go/pull/1609/files#diff-97eaea7067d7811544da444ef9a815e8R351

If you're okay with that, feel free to acknowledge and I will approve this PR.

Reply:
Yep, that's what I mean. Thanks for confirming!